fixes #114

hiway opened the pull request:
- [x] Explain the feature: fixes the issue described in #114
- [x] Read [CONTRIBUTING.md](https://github.com/iocage/iocage/blob/master/CONTRIBUTING.md)

`releases` is a list and `.split(".")` is not needed; removing it resolves #114 and `iocage fetch` works as expected.

hiway committed:
fixes #114

skarekrow commented:
Unfortunately doing so also will remove sortable releases from `list`, `list -l` and `list -r`. 

So a try block would be more appropriate here to catch that exception and remove the extra split in that case. 

Or a check for type and not do it if a list.

hiway commented:
Thank you. 

If I understand correctly, in `sort_release()`, `releases` may be either a `str` or a `list`.
Modifying 3 lines below line 238 should then work with both strings and lists?

```
    else:
        if list_sort:  # L:238
            if isinstance(releases, str):
                releases = releases.split(".")
            if len(releases[0]) < 2:
                releases = f"0{releases}"
            
            return releases
```

Is reassigning `releases = releases.split(".")` a cause for concern?

skarekrow commented:
I don't believe it would be a concern, I think that solution will work just fine. 

If it does I can fix it up later! Thanks for the PR :)

hiway committed:
`releases` may be string or list, handle both cases.

hiway commented:
Thank you for your work!
